package.json

```diff
@@ -7,6 +7,7 @@
         "start": "vite",
```

Comment:
Slik var det før: 
`"scripts": {
        "start": "cross-env REACT_APP_MOCK=true craco start",
        "build:dev": "cross-env REACT_APP_DEPLOYMENT_ENV=development craco build",
        "build:prod": "cross-env REACT_APP_DEPLOYMENT_ENV=production craco build",
        "build:mock": "cross-env PUBLIC_URL=/veilarbvisittkortfs REACT_APP_MOCK=true craco build",
        "test": "echo not implemented",
        "prettier": "prettier --write 'src/**/*.ts{,x}'",
        "lint": "eslint src"
    },`

Burde og sa npm run start ha NODE_MODE=mock?